docs/src/solutions.md

```diff
@@ -128,6 +128,22 @@ end
 # TODO: How to accurately measure the solve time.
 ```
 
+## Accessing MathOptInterface attributes
+
+[MathOptInterface](https://www.juliaopt.org/MathOptInterface.jl/stable/) defines a large
+number of model attributes that can be queried. Examples include
```

Comment:
We should eventually provide JuMP functions to expose these. Maybe open an issue?